doc-files/release-notes-17.0.2.md

```diff
@@ -0,0 +1,31 @@
+# Release Notes for JavaFX 17.0.2
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 17.0.2 release. As such, they complement the [JavaFX 17 Release Notes](https://github.com/openjdk/jfx/blob/jfx17/doc-files/release-notes-17.md).
+
+JavaFX 17.0.2 requires JDK 11 or later.
+
+## List of Task
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8272638](https://bugs.openjdk.java.net/browse/JDK-8272638)|Update copyright header for files modified in 2021|other
+[JDK-8274413](https://bugs.openjdk.java.net/browse/JDK-8274413)|FX: Update copyright year in docs, readme files to 2022|other
+[JDK-8279396](https://bugs.openjdk.java.net/browse/JDK-8279396)|Define version in .jcheck/conf|other

```

Comment:
We normally exclude `Task`s along with build and test changes from the release notes.

Reply:
I don't see a comment asking about it, but yes, your understanding is correct. The release notes should include bugs and enhancements (or backports of same) without build or test labels.